src/vanilla/Templates/MethodJsonPipelineTemplate.cshtml

```diff
@@ -4,8 +4,28 @@
 // Send Request
 let operationRes: msRest.HttpOperationResponse;
 try {
+@if (Model.RequestBody != null)
+{
+<text>
+  @(Model.ConstructRequestBodyMapper)
+</text>
+}
   operationRes = await client.sendOperationRequest(httpRequest, {
+@if (Model.RequestBody == null)
+{
+<text>
     httpMethod: "@(Model.HttpMethod.ToString().ToUpper())"
+</text>
+}
+else
+{
+<text>
+    httpMethod: "@(Model.HttpMethod.ToString().ToUpper())",
+    requestBodyMapper: requestModelMapper,
+    requestBodyName: "@(Model.RequestBody.Name)",
+    isXML: @(Model.CodeModel.ShouldGenerateXmlSerialization.ToString().ToLower())
```

Comment:
I know it makes the template more of a pain but can we include this only when true for size? (This might get into the territory where it's best to create a class with these fields on it and serialize it to JSON--can a serializer be convinced to do what I'm describing?)

Also doesn't strictly need to happen in this PR.

Reply:
Done in https://github.com/Azure/autorest.typescript/pull/92/commits/db0b08168255c86314c7c842ee0d8f90ba2c2703.